streampark-console/streampark-console-service/src/test/java/org/apache/streampark/console/core/service/YarnQueueServiceTest.java

```diff
@@ -115,13 +120,11 @@ void testCheckYarnQueue() {
         YarnQueue yarnQueue = mockYarnQueue(1L, "queue@");
         ResponseResult<String> result = yarnQueueService.checkYarnQueue(yarnQueue);
         assertThat(result.getStatus()).isEqualTo(2);
-        assertThat(result.getMsg()).isEqualTo(ERR_FORMAT_HINTS);
```

Comment:
We should not remove this since this is to ensure that the presentation is normal. We should adapt it.